app/battle_preparations/prepare_to_battle.py

```diff
@@ -0,0 +1,21 @@
+from app.knight.knight_class import Knight
+
+
+def preparations(knights: dict) -> list:

```

Comment:
Rename the function name, it should use verbs and represent action. You could use the name `prepare` instead.